docs/intro/scriptable_nodes.md

```diff
@@ -0,0 +1,12 @@
+# Python Script Operator
+
+In order to prototype your workflow or perform specific operations that are not included in Shift's built-in catalogs, you can script your custom operator with Python.  
+The **Python Script** operator offers the possibility to write your own code to tailor the processing of your data the way you want.  
+
```

Comment:
We should maybe give some more explanation to how the python script works. The name of input plugs are used as variable in the python script. And for each out plug the user should set a variable that has same name as the plug name

Reply:
So maybe a section dedicated to how custom plugs work? I thought this could go outside the Getting Started section seems like an advanced way to use PythonScript